graviti/dataset/dataset.py

```diff
@@ -196,5 +237,54 @@ def upload(
 
         """
 
+    def load(self) -> None:
+        """Load the dataset from Graviti."""
+        self._data = {}
+        response = list_segments(
+            self._url,
+            self._access_key,
+            self._dataset_id,
+            draft_number=self._status.draft_number,
+            commit=self._status.commit_id,
+        )
+        for sheet in response["segments"]:
+            sheet_name = sheet["name"]
+            data_details = list_data_details(
+                self._url,
+                self._access_key,
+                self._dataset_id,
+                sheet_name,
+                draft_number=self._status.draft_number,
+                commit=self._status.commit_id,
+            )
+
+            def factory_getter(
+                offset: int, limit: int, sheet_name: str = sheet_name
+            ) -> Dict[str, Any]:
+                return list_data_details(
+                    self._url,
+                    self._access_key,
+                    self._dataset_id,
+                    sheet_name,
+                    draft_number=self._status.draft_number,
+                    commit=self._status.commit_id,
+                    offset=offset,
+                    limit=limit,
+                )
+
+            factory = LazyFactory(
+                data_details["totalCount"],
+                128,
+                factory_getter,
+            )
+            # catalog = get_catalog(
```

Comment:
remove the useless comments

Reply:
this will be used after `catalog_to_schema` lands